ui/src/components/data-agent/ConnectorIdentified.tsx

```
@@ -0,0 +1,26 @@
+import { FC } from 'react';
+
+interface ConnectorIdentifiedProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)